### PR TITLE
Register with a GP practice tweaks

### PR DESCRIPTION
**Pull request by Demotive**
- Tweaked layout and copy in "nearest GP practices"
- Layout tweaks in practice pages
- Add email address and NHS no. fields to personal details
- More Purple Form grappling in "help us trace your medical records"
- Add "dispensing medicines and appliances" step
- Split donor questions into two steps

### CODE REVIEW

File: app/views/register-with-a-gp/register-without-signin.html [hunk 1]
```diff
@@ -30,12 +30,41 @@ <h1 class="heading-xlarge">
...
+        id: 'telephone',
+        label: 'Telephone number (optional)'
+      })
+    }}
+
+    <div class="form-group">
+      <label class="form-label-bold" for="nhs-number">
+        NHS number (optional)
+        <span class="form-hint">
+          If you know it &mdash; for example 943 476 5919
```
Comment (evilstreak) [inline app/views/register-with-a-gp/register-without-signin.html:59]: Is this NHS number a known example? If it's just random numbers, we should change it to something that we know to be not real.

File: app/views/register-with-a-gp/donor-organs.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+        <div class="form-group">
+          <input type="checkbox" id="pancreas">
+          <label class="form-control-checkbox" for="pancreas">
+            Pancreas
+          </label>
+        </div>
+
+        <div class="form-group">
+          <input type="checkbox" id="any-part-of-my-body">
```
Comment (evilstreak) [inline app/views/register-with-a-gp/donor-organs.html]: Should we pre-select this option?

File: app/views/register-with-a-gp/register-without-signin.html (hunk 1 above)
Comment (Demotive) [inline app/views/register-with-a-gp/register-without-signin.html:59]: http://systems.hscic.gov.uk/nhsnumber/patients/yournumber

File: app/views/register-with-a-gp/register-without-signin.html [hunk 3]
```diff
@@ -30,12 +30,41 @@ <h1 class="heading-xlarge">
...
     }}
 
-    {% include "includes/forms/gp-practice.html" with {
-      label: "Current GP practice or doctor"
-    } %}
+    {{
+      form_macros.address_fields({
+        id: 'address-1',
+        label: 'Address',
+        hint: 'As you’d write it on a envelope'
```
Comment (evilstreak) [inline app/views/register-with-a-gp/register-without-signin.html]: Typo: **an** envelope

File: app/views/register-with-a-gp/register-without-signin.html (hunk 1 above)
Comment (evilstreak) [inline app/views/register-with-a-gp/register-without-signin.html:59]: :+1: Excellent

**Comment by evilstreak** [comment]
This looks great. Love the new buttons.

File: app/views/register-with-a-gp/donor-organs.html (hunk 2 above)
Comment (Demotive) [inline app/views/register-with-a-gp/donor-organs.html]: We should review the overall content with Dr Marcus. But yes, we'll preselect this, good point.

File: app/views/register-with-a-gp/register-without-signin.html (hunk 3 above)
Comment (Demotive) [inline app/views/register-with-a-gp/register-without-signin.html]: :|

File: app/views/register-with-a-gp/donor-organs.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        No
+      </label>
+    </fieldset>
+
+    <div class="js-hidden panel-indent" id="organ-donor">
+      <fieldset>
+        <legend>I will donate</legend>
+
+        <div class="form-group">
+          <input type="checkbox" id="any-part-of-my-body" checked="checked">
```
Comment (Demotive) [inline app/views/register-with-a-gp/donor-organs.html:35]: @evilstreak there's probably more thinking involved here... for the time being let's roll with this. But we need to get to a point where the user is saying either:

"I'm happy to donate any part of my body"
OR
"I am ONLY happy for you to use the following organs"

...which basically seems odd. We should find out the reasoning behind this.

Comment (evilstreak) [inline app/views/register-with-a-gp/donor-organs.html:35]: I agree!

**Pull request merged**